libs/application/templates/aosh/transfer-of-machine-ownership/src/fields/MachinesField/MachineSelectField.tsx

```diff
@@ -0,0 +1,159 @@
+import { FieldBaseProps, Option } from '@island.is/application/types'
```

Comment:
This component is calling graphql on select and we just had a meeting with Þorri and that is not possible with the shared selectField component, but if you have a solution that would be nice because we are doing this in multiple applications

Reply:
Its on the to do list to be able to use the template api for async calls. With that we could look into making components that can support this behavior.